shell/shared/grain.js

```diff
@@ -126,13 +124,6 @@ if (Meteor.isServer) {
       promise.cancel();
     });
   });
-
-  function cleanupExpiredTokens() {
```

Comment:
The "expired" case is different because then the user might actually want to request access.